### ISSUE TITLE
Additive waypoint editing

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/cac-tripplanner#502]
Depends on #499.

Rather than passing all turn points in route to Leaflet Draw on entering edit mode, initially only display journey start and end point (connected by a straight line). When user releases mouse after dragging route line, store the added waypoint and immediately re-query for the edited route. Entering edit mode again from a route with one or more waypoints will pass the previously added waypoints to Leaflet Draw, in addition to the start and end points.

Do not display the Leaflet Draw edit toolbar (edit pencil button in upper left of map, with 'save' and 'cancel' buttons to its right).

**Comment by flibbertigibbet** [comment]
Closing in favor of #514.
